docker/vast/Dockerfile

```diff
@@ -0,0 +1,167 @@
+# -- dependencies --------------------------------------------------------------
```

Comment:
It seems the dockerfile is duplicated, isn't it?

Reply:
I think this slipped back in during a rebase. 🙈